recipes-bsp/u-boot/u-boot_2015.10.bbappend

```diff
@@ -0,0 +1,14 @@
+# Keep this separately from the rest of the .bb file in case that .bb file is
+# overridden from another layer.
+
+require u-boot-mender.inc
+
+# Copy script to the deploy area with u-boot, uImage, and rootfs
+do_deploy_uenv () {
+    if [ -e ${WORKDIR}/uEnv.txt ] ; then
+        install -d ${DEPLOY_DIR_IMAGE}
+        install -m 0444 ${WORKDIR}/uEnv.txt ${DEPLOY_DIR_IMAGE}
```

Comment:
Yup. This is a part of u-boot and later is copied to boot partition. I don't know any other easy way to do this. What is more this will be needed when you will copy files manually to boot partition.